README.md

```diff
@@ -1,48 +1,48 @@
-This is an LSSTC-supported project, to form a small community committed to learning how to use, and explain, the LSST DM Stack. 
+This is an LSSTC-supported project, to form a small community committed to learning how to use, and explain, the LSST software (DM) Stack. 
```

Comment:
"LSST software data management (DM) stack"

Reply:
Good, I like this. Changed!